tools/mentors/view_page_text_image.php

```diff
@@ -6,6 +6,10 @@
 
 require_login();
 
+$projectid = array_get($_GET, "project", "");
+$page = array_get($_GET, "imagefile", "");
```

Comment:
This should use `validate_page_image_filename()` (although my MR changes this to `get_page_image_param()`).